Backend/router/v1/report.ctrl.js

```diff
@@ -1,41 +1,70 @@
 const models = require('../../models');
 
 const createReport = (req, res) => {
-  models.report.create({
-    what: req.body.data[0].what,
-    location: req.body.data[0].location,
-    time: req.body.data[0].time,
-    who: req.body.data[0].who,
-    details: req.body.data[0].details,
-  }).then((result) => {
-    res.status(201);
-    res.send("good");
-  });
-}
+  if(req.isAuthenticated()){
+    console.log("User id : ", req.user[0].dataValues.id);
```

Comment:
Please remove all console.log that you used for testing purpose

Reply:
If you added the log for some other reason, please add a comment for your justification